juha/programmers/4/완주하지-못한-선수.js

```diff
@@ -0,0 +1,16 @@
+function solution(participant, completion) {
+  const athletes = {};
+  for (let name of participant) {
+    athletes[name] = athletes[name] ? athletes[name] + 1 : 1;
+  }
```

Comment:
배열의 요소를 순서대로 순환하면서 기능 (함수) 를 수행하는 방법으로는 배열 매서드를 추천합니다. 로직은 변함이 없고 `for 문` >> `forEach` 으로 사용 키워드만 변경했습니다.
```suggestion
  participant.forEach(name => {
    athletes[name] = athletes[name] ? athletes[name] + 1 : 1;
  });
```